recipes/libpdal/build.sh

```diff
@@ -52,11 +52,12 @@ if [ $(uname) == Darwin ]; then
     export CXXFLAGS="${CXXFLAGS} -stdlib=libc++"
     export CPPFLAGS="-I/usr/local/opt/curl/include"
     export LDFLAGS="${LDFLAGS} -headerpad_max_install_names"
-    CMAKE_ARGS="-DCURL_INCLUDE_DIR=/usr/local/opt/curl/include -DCURL_LIBRARY=/usr/local/opt/curl/lib/libcurl.dylib"
+    CMAKE_ARGS="-DCURL_INCLUDE_DIR=/usr/local/opt/curl/include -DCURL_LIBRARY=/usr/local/opt/curl/lib/libcurl.dylib -DGDAL_LIBRARY:PATH=$PREFIX/lib/libgdal.dylib"
 else
     OPTS="--disable-rpath"
     COMP_CC=gcc
     COMP_CXX=g++
+    export LDFLAGS="${LDFLAGS} -lkea -lproj -lpoppler -lxerces-c-3.2 -ldap -lpng16 -lpcre -liconv -lhdf5 -lgeos-3.6.2 -licui18n -licuuc -licudata -lcom_err -lhdf5_hl -lhdf5_cpp -lminizip -luriparser -lkmlbase -lkmldom -lkmlengine -ldapclient -lssl -lssh2 -lcrypto"
```

Comment:
this does not look very encouraging.

Reply:
@danlipsa is this something referred in the pdal docs or this is just from the experience.